ui/job-view/headerbars/PrimaryNavBar.jsx

```diff
@@ -23,7 +24,11 @@ export default function PrimaryNavBar(props) {
       <div id="th-global-top-nav-panel">
         <nav id="th-global-navbar" className="navbar navbar-dark">
           <div id="th-global-navbar-top">
-            <LogoMenu />
+            <LogoMenu
+              menuText="Treeherder"
+              menuTitle="Treeherder services"
```

Comment:
Perfherder also uses the title "Treeherder Services":
https://github.com/mozilla/treeherder/blob/41164215feb9a1d327f77408c3171eb018159ea5/ui/perf.html#L20

Should it be moved into the component and be hardcoded, instead of being a prop? This would also reduce the chance that people accidentally invert the two prop meanings (menuTitle and menuText) - which would break the `choices` filtering.

Reply:
Sure, that makes sense.